session/session.go

```diff
@@ -1353,6 +1353,10 @@ func (s *session) Close() {
 	if s.statsCollector != nil {
 		s.statsCollector.Delete()
 	}
+	bindValue := s.Value(bindinfo.SessionBindInfoKeyType)
```

Comment:
is it a bugfix?

Reply:
No, it is required by this PR, we need to remove all the metrics about the closed session.